docs/Antora/modules/getting_started/pages/quick_start_application.adoc

```diff
@@ -254,26 +244,26 @@ instance = opendaq.Instance()
 
 # Find and output the names and connection strings of all available devices
 for device_info in instance.available_devices:
-    print("Name:", device_info.name, "Connection string:", device_info.connection_string)
+    print('Device name: "{}". Connection string: "{}"'.format(device_info.name, device_info.connection_string))
     for capability in device_info.server_capabilities:
-        print("Capability:", capability.protocol_name, "Connection string:", capability.connection_string)
+        print(' - Supports protocol "{}" by connection string: "{}"'.format(capability.protocol_name, capability.connection_string))
 ----
 ====
-Device info connection string for most of devices has format `daq://[manufacturer]_[serial number]` and server capabilities which stores supporting protocols.
 Now run the simulator and the above client code. We should see the console output several Device names and connection
 strings. 
 
 * `Device 0 : daqref://device0`
-* `Device 1 : daq://openDAQ_dev_ser_1`
-** `openDAQ OpcUa : daq.opcua://xxx.xxx.xxx.xxx/`
+* `Device 1 : daq://openDAQ_xxx`
 ** `openDAQ Native Streaming : daq.ns://xxx.xxx.xxx.xxx/`
+** `openDAQ Native Configuration : daq.nd://xxx.xxx.xxx.xxx/`
+** `openDAQ OpcUa : daq.opcua://xxx.xxx.xxx.xxx/`
 
-(the `daq.*://` addresses are specific to your simulator instance). We can use their corresponding connection strings to add / connect to the devices.
+In the above example, we see that the simulator device is shown as `daq://openDAQ_xxx` where `openDAQ` is manufacturer name and `xxx` is a serial number.
+Because simulator was discovered by multiple modules, information about simulator was grouped. To connect to the simulator, we can use any of the provided connection strings.
```

Comment:
Can we add that we prefer using `daq.nd://` to avoid users trying to connect with only streaming or only configuration?

Reply:
Ah I see `daq://` is used bellow but it should be explained what is the difference between any of the `daq.xyz://` and only `daq://`.